libbpf.yaml

```
@@ -0,0 +1,39 @@
+package:
+  name: libbpf
+  version: 1.1.0
+  epoch: 0
+  description: "A library for interacting with the Linux kernel's Berkeley Packet Filter (BPF) facility from user space"
+  copyright:
+    - license: GPL-2.0-only
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - elfutils-dev
+      - linux-headers
+      - zlib-dev  
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/libbpf/libbpf
+      tag: v${{package.version}}
```

maybe need to drop the "v" here?
```suggestion
      tag: ${{package.version}}
```